codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,7 +195,14 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the flow controllers.
+            connection.local().flowController(new DefaultHttp2LocalFlowController(connection, writer,
```

Comment:
@Scottmitch ah I think I had misread your comment.  Should we just remove the defaults from encoder/decoder?